ReactiveCocoa/Swift/Action.swift

```diff
@@ -202,18 +202,18 @@ public final class CocoaAction: NSObject {
 	}
 }
 
-/// The type of error that can occur from Action.apply, where `E` is the type of
+/// The type of error that can occur from Action.apply, where `Error` is the type of
 /// error that can be generated by the specific Action instance.
-public enum ActionError<E: ErrorType>: ErrorType {
+public enum ActionError<Error: ErrorType>: ErrorType {
 	/// The producer returned from apply() was started while the Action was
 	/// disabled.
 	case NotEnabled
 
 	/// The producer returned from apply() sent the given error.
-	case ProducerError(E)
+	case ProducerError(Error)
 }
 
-public func == <E: Equatable>(lhs: ActionError<E>, rhs: ActionError<E>) -> Bool {
+public func == <Error: Equatable>(lhs: ActionError<Error>, rhs: ActionError<Error>) -> Bool {
```

Comment:
Not for normal functions, but for operators I find it really helps with readability.